TRENZ.Docs.API/Program.cs

```diff
@@ -38,7 +38,8 @@
     builder.Services.AddAuthAdapter();
 }
 
-builder.Services.AddHostedService<Worker>();
+builder.Services.AddSingleton<Worker>();
+builder.Services.AddHostedService<Worker>(sp => sp.GetRequiredService<Worker>());
```

Comment:
No. The worker needs to be made available in the DI container. The `AddHostedService` doesn't make it available for retrieval, it just tells the Host which service to run. Since we only want one single instance of the worker, I register it separately (so the API endpoint can retrieve it from DI) and then return this specific instance as a hosted service when requested.